ext/standard/tests/array/locale_sort.phpt

```diff
@@ -6,7 +6,7 @@ if (substr(PHP_OS, 0, 3) == 'WIN') {
   die("skip Unix locale name only, not available on windows (and crashes with VC6)\n");
```

Comment:
Looks like Windows support can be improved as well. I'll follow up after this PR has been merged.